README.md

```diff
@@ -561,6 +562,34 @@ In the above example, the directory structure of the archive would be as follows
 /<archive>/README.md
 ```
 
+You can use the `bin-leading-dir` option to create extra leading directory(s) for `bin` (e.g. the binary file).
```

Comment:
> e.g. the binary file

Did you mean "i.e."?

Reply:
emmmmmmm, my google translator says: "e.g." is like "for example". while it shows nothing about "i.e.", so you can make decision about it .

Reply:
If the only thing that matches `bin` here is a binary file(s), then I think "i.e.," (in other words) is appropriate. Otherwise, "e.g.," (for example) is appropriate.

Well, I feel something like "leading directory(s) for binary file(s) specified by `bin` option" would be easier to understand, though.

Reply:
Update: update `description` to: `Leading directory(s) for binary file(s) specified by `bin` option`.